frontend/src/main-components/InsideFolderContent.js

```diff
@@ -1,23 +1,72 @@
-import React from "react";
-import styled from "@emotion/styled";
+import React from 'react';
+import styled from '@emotion/styled';
+import Flashcard from '../flashcards/Flashcard.js';
+import NewCard from '../flashcards/NewCard.js';
+import { LogDebugMessage } from './LogDebugMessage.js';
 
+const ContainerStyle = {
+  flex: "9",
+  display: "flex",
+  border: "1rem solid white",
+  borderRadius: "1rem",
+  backgroundColor: "white",
+  margin: "1%",
+  flexDirection: "column",
+};
 
-const InsideFolderContent = (props) => {
-  const Container = styled.div`
-    flex: 9;
-    display: flex;
-    border: 1rem solid white;
-    border-radius: 1rem;
-    background-color: white;
-    margin: 1%;
-  `;
+const CardContainer = {
+  display: 'flex',
+  flexFlow: 'row wrap',
+  justifyContent: 'space-around',
+  alignItems: 'baseline',
+};
 
+class InsideFolderContent extends React.Component {
+  constructor(props) {
+	super(props);
+	this.state = {
+	  isDataFetched: false,
+	  flashcards: [],
+	};
+  }
 
-  return (
-    <Container>
-      <h1>Your Folder has 0 cards</h1>
-    </Container>
-  );
-};
+  componentDidMount() {
+	let flashcards=[];
+	const FetchPromise = fetch(`/usercards?folderKey=${this.props.folderKey}`, { method: 'GET' });
```

Comment:
Uppercase should limited to class names: https://google.github.io/styleguide/jsguide.html#naming-class-names

Reply:
fixed!